components/script/dom/window.rs

```diff
@@ -330,6 +336,29 @@ impl Window {
     pub fn webvr_thread(&self) -> Option<IpcSender<WebVRMsg>> {
         self.webvr_thread.clone()
     }
+
+    #[allow(unrooted_must_root)]
```

Comment:
I'm pretty sure our static analysis gets confused by the use of the entry API here - we end up with `Entry<JS<Node>>` IIRC.